hnn_core/dipole.py

```diff
@@ -275,5 +275,5 @@ def write(self, fname):
             Full path to the output file (.txt)
         """
         X = np.r_[[self.t, self.dpl['agg'], self.dpl['L2'], self.dpl['L5']]].T
-        np.savetxt('dpl2.txt', X, fmt=['%3.3f', '%5.4f', '%5.4f', '%5.4f'],
+        np.savetxt(fname, X, fmt=['%3.3f', '%5.4f', '%5.4f', '%5.4f'],
```

Comment:
can you also add a test for this? Something like:

```py
assert op.exists(fname)
```

This way we won't have a regression of the bug